src/runners/RunnerUtil.ml

```diff
@@ -107,17 +107,15 @@ let parse_cli () =
     ("-jsonerrors", Arg.Unit (fun () -> r_json_errors := true), "Print errors in JSON format");
   ] in 
 
-  let mandatory_usage = " -libdir /path/to/stdlib input.scilla" in
-  let optional_usage = String.concat "\n\t"
-    (List.map (function (flag,_,desc) -> flag ^ "\t" ^ desc) speclist) in
-  let usage = mandatory_usage ^
-    "\nFull list of CLI flags:\n\t" ^
-    optional_usage in
+  let mandatory_usage = "Usage:\n" ^ Sys.argv.(0) ^ " -libdir /path/to/stdlib input.scilla\n" in
+  let optional_usage = String.concat "\n  "
+    (List.map (function (flag,_,desc) -> flag ^ "  " ^ desc) speclist) in
```

Comment:
This must be an artifact from a typo on my part but the two spaces here are a little inconsistent with the single space in https://github.com/Zilliqa/scilla/pull/436/commits/86cec1c7e934ff514ca3b9f662e8ef203e3e7649#diff-90619754eb57d32adb56bdc0ea4960bcR141

Reply:
I've now fixed this.